pkgs/by-name/tg/tgpt/package.nix

```diff
@@ -1,26 +1,29 @@
 { lib
 , buildGoModule
 , fetchFromGitHub
+, nix-update-script
```

Comment:
Is usually not as the bot will pick-up new releases automatically.

Reply:
If the bot picked up the release without setting `passthru.updateScript`, I wouldn't have started this PR :)

Also, this is to make it easier for contributors to bump the version, in case the bot's build fails. Which happened in this case, some tests are failing that I'm investigating a bit.

Reply:
> If the bot picked up the release without setting passthru.updateScript, I wouldn't have started this PR :)

The bot didn't create an update because of the failing tests. `nix-update` works usually fine for manual updates.